JULIERAJ/KIDS-FIRST-Front
# Feature/72/move logout to separate component

## One Line Description
move logout to separate component

## Test Steps
1. Open the application.
2. Navigate to the sidebar where the logout button is displayed 
3. Verify that the logout button appears in the same location and with the same styling as before the refactor.
4. Ensure that the overall layout of the sidebar has not been affected by the changes.
5. Click the logout button in the sidebar.
6. Check that the user is logged out and redirected to the login page.

## UI/UX Outcome
![logout component](https://github.com/user-attachments/assets/87a0ee3f-5177-44c9-a2a1-db7581516577)


## Checklist
